mvc/2.1/_index.md

```diff
@@ -1,23 +1,59 @@
 ---
-title: "Jakarta MVC 2.1 (under development)"
+title: "Jakarta MVC 2.1"
 date: 2021-04-15
 summary: "Release for Jakarta EE 10"
```

Comment:
I don't think it relates Jakarta EE 10 release.

Reply:
You're right. What we meant by the comment is that it requires EE 10. 
I can change the text to "Release that supports Jakarta EE 10" after the review.